libs/application-generic/src/services/auth/shared.ts

```diff
@@ -48,6 +48,10 @@ export const buildOauthRedirectUrl = (request): string => {
   if (invitationToken) {
     url += `&invitationToken=${invitationToken}`;
   }
+  const isLoginPage = JSON.parse(request.query.state).isLoginPage;
```

Comment:
Instead of marking whether OAuth comes from loginPage, we can add a `redirect_url` parameter set from the front end, which would simplify the FE logic.

Reply:
@SokratisVidros which means instead of hardcoed `/auth/login `at [this](https://github.com/novuhq/novu/blob/fix/hubspot-ques-form-revert/libs/application-generic/src/services/auth/shared.ts#L2) line, `redirect_url` query param value should be used?